event-exporter/Makefile

```diff
@@ -28,10 +28,23 @@ test:
 	${ENVVAR} go test -mod=vendor ./...
 
 container:
-	docker build --pull -t ${PREFIX}/${IMAGE_NAME}:${TAG} .
+	docker buildx build . --pull -t ${PREFIX}/${IMAGE_NAME}:${TAG} --load
+
+container-multiarch:
+	docker buildx build . --pull -t ${PREFIX}/${IMAGE_NAME}-amd64:${TAG} --platform=linux/amd64 --load
```

Comment:
Suggestion: Have a variable called ALL_ARCH and build containers for ALL_ARCH in stead of hard coding. With that I think you can just reuse the `container` target.

Reply:
Updated the build targets to reference architecture in a more extensible way